src/Docker/Runner/DataLoader/ABSWorkspaceProvider.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Keboola\DockerBundle\Docker\Runner\DataLoader;
+
+class ABSWorkspaceProvider extends AbstractWorkspaceProvider
```

Comment:
Tu ma teda napadlo, ze ci tych WorkspaceProviderov, pripadne cely DataLoader nevytiahnut do vlastnej kniznice? Ta by sa potom dala pouzit v testoch IM/OM minimalne.

Reply:
jojo, uz se to tam zacina rysovat